src/apps/sources/pages/CreateSourcePage.tsx

```diff
@@ -25,6 +26,7 @@ interface Props {
     ) => void;
     loading: boolean;
     newSource?: APISource;
+    resetCreateSourceAction: () => void;
```

Comment:
This name should be the same as the name used in `mapActionsToProps` below, i.e., `resetCreateSource`.